orchagent/qosorch.cpp

```diff
@@ -1590,9 +1590,19 @@ bool QosOrch::applySchedulerToQueueSchedulerGroup(Port &port, size_t queue_ind,
     attr.value.oid = scheduler_profile_id;
 
     sai_status = sai_scheduler_group_api->set_scheduler_group_attribute(group_id, &attr);
+    if (SAI_STATUS_NOT_SUPPORTED != sai_status)
```

Comment:
Suggest using
```
        if (SAI_STATUS_IS_ATTR_NOT_SUPPORTED(status) || SAI_STATUS_IS_ATTR_NOT_IMPLEMENTED(status)
                 || status ==  SAI_STATUS_NOT_SUPPORTED || status == SAI_STATUS_NOT_IMPLEMENTED)
```
Refer https://github.com/sonic-net/sonic-swss/blob/master/orchagent/switchorch.cpp#L678 as an example.

Reply:
Thank you for reviewing. This condition "SAI_STATUS_NOT_SUPPORTED != sai_status" was not right in the first place. Will fix this.